middleware/http/wasm/httpwasm.go

```diff
@@ -20,6 +20,8 @@ import (
 	"github.com/dapr/kit/logger"
 )
 
+const metadataConfig = "config"
```

Comment:
can this be a typed value vs a string lookup? In any case, possibly deserves a comment like `config is the guest configuration, assigned during instantiation by handler.GuestConfig`

Reply:
Added to metadata struct, also rename config to guestConfig.